### PR TITLE
Grpc tests

### PR DESCRIPTION
**Pull request by afanasievM**

### CODE REVIEW

File: transaction-service/src/test/kotlin/ua/com/foxminded/restClient/gprc/TransactionGRPCServiceTest.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.mockito.kotlin.eq
+import org.springframework.data.domain.Pageable
+import reactor.core.publisher.Flux
+import reactor.core.publisher.Mono
+import reactor.test.StepVerifier
+import ua.com.foxminded.restClient.service.CurrencyExchangeService
+import ua.com.foxminded.restClient.service.TransactionService
+import utils.Transactions
+
+class TransactionGRPCServiceTest {
```
Comment (ajax-shulepa-a) [inline transaction-service/src/test/kotlin/ua/com/foxminded/restClient/gprc/TransactionGRPCServiceTest.kt]: write an integration test for gRPC service, please as it's entry point for clients

File: transaction-service/build.gradle.kts [hunk 2]
```diff
@@ -17,7 +17,7 @@ plugins {
     id("org.jetbrains.kotlin.kapt") version "1.8.21"
     id("org.jetbrains.kotlin.plugin.allopen") version "1.8.21"
     id("io.gitlab.arturbosch.detekt") version "1.23.0"
-
+    `java-test-fixtures`
```
Comment (ajax-shulepa-a) [inline transaction-service/build.gradle.kts:20]: good choice

Comment (afanasievM) [reply transaction-service/build.gradle.kts:20]: 👌

File: transaction-service/src/test/kotlin/ua/com/foxminded/restClient/gprc/TransactionGRPCServiceTest.kt (hunk 1 above)
Comment (afanasievM) [reply transaction-service/src/test/kotlin/ua/com/foxminded/restClient/gprc/TransactionGRPCServiceTest.kt]: done

**Review by ajax-shulepa-a — APPROVED** [review]

**Pull request merged**